app.rb

```diff
@@ -0,0 +1,75 @@
+require 'sinatra'
+require "sinatra/reloader" if development?
+require 'sinatra/flash'
+require 'uri'
+require './lib/peep'
+require './lib/user'
+
+class Chitter < Sinatra::Base
+  enable :sessions, :method_override
+  register Sinatra::Flash
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  # get '/' do
+  #   erb :home
+  # end
+  get '/peeps' do
+    @user = User.find(id: session[:user_id])
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/registrations/sign_up' do
+    erb :'registrations/sign_up'
+  end
+
+  post '/registrations' do
+    @user = User.create(id: params[:id], first_name: params[:first_name], 
+last_name: params[:last_name], email: params[:email], user_password: params[:user_password])
+    session[:user_id] = @user.id
+    session[:user_name] = @user.first_name
+    redirect '/peeps'
+  end
+
+  get '/sessions/login' do
+    erb :'sessions/login'
+  end
+
+  post '/sessions' do
+    @user = User.find_by(email: params['email'], user_password: params['user_password'])
+    if @user
+      session[:user_id] = @user.id
+      session[:user_name] = @user.first_name
+      redirect '/peeps'
+    else
+      redirect '/sessions/login'
+    end
```

Comment:
Nice clean and readable code here in your routes. I like your use of find_by to find the correct user and ensure the correct information is being stored. Overall great use of instance variables and sessions to save user input.